create rare disease subset documentation (#6403)
monarch-initiative/mondo: docs/editors-guide/rare-disease-subset.md

# Rare Disease Subset in Mondo

A rare disease is a disease that affects fewer than 200,000 people in the United States. However, this definition is not universal, and it varies between rare disease sources, geographical locations, etc. Consequently, rare disease criteria also vary between sources, and one might have difficulties reconciling sources and creating a list for rare diseases. Mondo recognizes these impediments and differences of view. This document explains how Mondo handles rare diseases, how one could get a rare disease list, and the efforts made to bring the community together.

#### What are rare disease in Mondo?
We rely on rare disease authorities (see table below) to determine all diseases recognized as “rare”, according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annoted with `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority).

> example: calciphylaxis (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.

Reviewer: ```suggestion
Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
```

> ![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)

Reviewer: ```suggestion

![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)
```

Otherwise wrong line break.

In our [2021 publication](https://mondo.monarchinitiative.org/pages/analysis/), we identified rare diseases as 

> - terms annotated with the `‘bearer of’ some rare` axiom (as provided by Mondo contributing resources)

Reviewer: This is IMO too weak.. Is there any hope we can drop this phrase and replace it by the operational criteria that _led us_ to assert that axiom?

Author: We 'could' but these came before my time, and I don't know if there were any operational criteria... 

@nicolevasilevsky, do you know how these "bearer of rare" axiom were created? All I could find is that it was provided by the sources.

Author: Note to self: can also add this reference: https://pubmed.ncbi.nlm.nih.gov/32020066/

- **AND** limited to the ‘human disease’ branch (which excludes ‘susceptibility’ terms)
_(note that the "gard subset" annotation reported in the publication has been ignored here)_
The Mondo terms fulfilling the above conditions were annotated with the 'in_subset' annotation 'mondo_rare' (see arrows in image above).

#### The Mondo rare disease subset include ALL rare diseases according to ALL rare disease sources.
The Mondo rare disease subset represents the UNION of rare diseases according to all authority sources and Mondo. Most of the diseases in this subset will be viewed as “rare” by most authorities, but some will be viewed as “rare” by only some authorities.

Rare disease terms in Mondo are annotated with `in_subset: rare` (see arrowhead in image above), and include
- terms with the annotation in_subset ‘mondo_rare’
- **AND** terms with the annotation in_subset ‘X_rare’ (which includes ALL the rare disease authorities represented in Mondo)

> - **AND** limited to the ‘human disease’ branch (which excludes ‘susceptibility’ terms)

Author: note that this is not currently the case, but I will add a GH issue to update the sparql

#### Where to find the Mondo rare disease subset, and what is included in the file?
The Mondo rare disease can be found as a product of the Mondo release, available in .obo and .json formats
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo)
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json)

This mondo-rare subset includes **all rare diseases** (as defined above) AND their **ontological parents**.

Users can filter using the subset annotation “mondo_rare” to get a list of Mondo rare diseases without including their ontological parents.

Users can filter using the subset annotation “X_rare” (where X is a rare disease authoritative source) to get a list of "rare disease according to X".

#### Current rare disease authority sources represented in Mondo

| Rare disease authority | link | in_subset annotation in Mondo | Rare disease source criteria |
| --- | --- | --- | --- |
|NORD |(website) | nord_rare | *(coming soon)*|
|GARD |(website) | gard_rare | *(coming soon)*|
|Orphanet |(website) | orphanet_rare | *(coming soon)*|

> 

Author: still need to update the website link here

#### What are rare disease in Mondo?
We rely on rare disease authorities (see table below) to determine all diseases recognized as “rare”, according to their own criteria. When a disease is considered “rare” according to a source, the corresponding Mondo term is annoted with `in_subset` annotation expressing that this disease is considered rare according to this source: `X_rare`  (where X is the rare disease authority).

Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.

![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)

> Some Mondo terms were annotated with a `‘bearer of’ some rare` axiom. These axioms were added automatically to terms coming from Orphanet (before the creation of 'orphanet_rare' in_subset annotation), and manually by a curator based on the disease definition.

Reviewer: You only list Orphanet here as the source for "bearer of some rare". Why then not keep this in sync with Orphanet? The only thing that would matter if Mondo curators would add this restriction on a class that was not defined as rare according to anyone else. INn that case, we need to say what Mondo considers rare at all.

Here is an idea: we query for all classes that are rare according to mondo but no one else (https://api.triplydb.com/s/urdlrUxt_), and develop a short narrative for why these should be considered rare despite no one else saying so? I don't think this is worth doing until _after_ there GARD and NORD PRs are merged though.

Author: I don't know why we are not keeping it in sync with Orphanet... From my point of view, having "orphanet-rare" subset annotation and "bearer of rare" for orphanet is redundant (or would be redundant if we were keeping it in sync)
IMO, these axioms should be added to "every" rare diseases, or to only diseases that a curator identified as rare (and is not in a rare subset). 

I created an [issue](https://github.com/monarch-initiative/mondo/issues/6448) to discuss this.

In our [2021 publication](https://mondo.monarchinitiative.org/pages/analysis/) ([and here](https://pubmed.ncbi.nlm.nih.gov/32020066/)), we identified rare diseases as
- terms annotated with the `‘bearer of’ some rare` axiom (as provided by Mondo contributing resources)
- **AND** limited to the ‘human disease’ branch (which excludes ‘susceptibility’ terms)
_(note that the "gard subset" annotation reported in the publication has been ignored here)_

The Mondo terms fulfilling the above conditions were annotated with the 'in_subset' annotation 'mondo_rare' (see arrows in image above).

#### The Mondo rare disease subset include ALL rare diseases according to ALL rare disease sources.

> The Mondo rare disease subset represents the UNION of rare diseases according to all authority sources and Mondo. Most of the diseases in this subset will be viewed as “rare” by most authorities, but some will be viewed as “rare” by only some authorities.

Reviewer: This is currently not true. There are currently 1/3 of the classes only supported by Mondo!

Rare disease terms in Mondo are annotated with `in_subset: rare` (see arrowhead in image above), and include
- terms with the annotation in_subset ‘mondo_rare’
- **AND** terms with the annotation in_subset ‘X_rare’ (which includes ALL the rare disease authorities represented in Mondo)
- **AND** limited to the ‘human disease’ branch (which excludes ‘susceptibility’ terms)

#### Where to find the Mondo rare disease subset, and what is included in the file?
The Mondo rare disease can be found as a product of the Mondo release, available in .obo and .json formats
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo)
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json)

This mondo-rare subset includes **all rare diseases** (as defined above) AND their **ontological parents**.

> Users can filter using the subset annotation “mondo_rare” to get a list of Mondo rare diseases without including their ontological parents.

Reviewer: ```suggestion
Users can filter using the subset annotation “rare” to get a list of Mondo rare diseases without including their ontological parents.
```

Reviewer: mondo_rare is only bearer of some rare

Users can filter using the subset annotation “X_rare” (where X is a rare disease authoritative source) to get a list of "rare disease according to X".

#### Current rare disease authority sources represented in Mondo

| Rare disease authority | link | in_subset annotation in Mondo | Rare disease source criteria |
| --- | --- | --- | --- |

> |GARD |[https://rarediseases.info.nih.gov/](https://rarediseases.info.nih.gov/) | gard_rare | *(coming soon)*|

Reviewer: ```suggestion
| GARD | [https://rarediseases.info.nih.gov/](https://rarediseases.info.nih.gov/) | gard_rare | *(coming soon)*|
```

> |NORD |[https://rarediseases.org/](https://rarediseases.org/) | nord_rare | *(coming soon)*|

Reviewer: ```suggestion
| NORD | [https://rarediseases.org/](https://rarediseases.org/) | nord_rare | *(coming soon)*|
```

> |Orphanet |[https://www.orpha.net/](https://www.orpha.net/) | orphanet_rare | *(coming soon)*|

Reviewer: ```suggestion
| Orphanet | [https://www.orpha.net/](https://www.orpha.net/) | orphanet_rare | *(coming soon)*|
```

Example: 'calciphylaxis' (MONDO:0017215) is considered "rare" according to NORD, GARD, and Orphanet, as shown by the 'in_subset' annotations 'nord_rare', 'gard_rare', 'orphanet_rare'.
![rareexample](https://github.com/monarch-initiative/mondo/assets/12737987/a5dc4b6e-c3d1-4bca-926a-f81fdcea5ddb)

**2. manual curation**
In some occasional cases, some diseases are reported in the literature as 'rare', though they are not reported in the rare disease authorities' list. In these cases, we record the disease as rare by manually adding the "mondo_rare" subset annotation to the term. These manually curated annotations must be supported by the literature. This new rare disease is reported to the rare disease authorities. 

**3. ontology inference**
Ontological parents pass their characteristics (here: axioms) to their children. We leverage this ontological property for rare diseases: ontological children of a term that is considered "rare" according to the rules above are getting the "inferred_rare" subset annotation.

#### The Mondo rare disease subset includes ALL rare diseases according to ALL rare disease sources.

> The Mondo rare disease subset represents the UNION of rare diseases according to all authority sources, manual curation, and ontology inference. It should be noted that most of the diseases in this subset will be viewed as “rare” by _most_ authorities, but some will be viewed as “rare” by _only some_ authorities. 

Reviewer: Have you checked this with the subset report query? I seem to remember there were huge differences. We could also add an overlap plot here (venn diagram) to illustrate the overlap. You can make a separate issue if you would like that (not a blocker here).

Author: I haven't compared the different subsets and I agree having a venn diagram would help. Let's talk about how to do this, because it is not necessarily straight forward. I will make a separate issue.

Rare disease terms in Mondo are annotated with `in_subset: rare` (see arrowhead in image above), and include
- terms with the annotation in_subset ‘X_rare’ (which includes ALL the rare disease authorities represented in Mondo)
- terms with the annotation in_subset ‘mondo_rare’ (representing the manually curated rare diseases)
- terms with the annotation in_subset 'inferred_rare' (representing the diseases  ontologically infered as rare based on their parents)

Users should decide which subset they want to use for their applications. They can do so by filtering the rare disease subset by any of the 'in_subset' annotation. 

#### Where to find the Mondo rare disease subset, and what is included in the file?
The Mondo rare disease can be found as a product of the Mondo release, available in .obo and .json formats
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.obo)
- [http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json](http://purl.obolibrary.org/obo/mondo/subsets/mondo-rare.json)

#### What happens when a rare disease term is obsoleted or lose its "rare" characteristic?

> Terms in Mondo can be obsoleted for multiple reasons (see obsoletion documents here). It is possible that some of these terms to be obsoleted are in the rare disease subset. 

Reviewer: Missing link to obsolescence docs

> 1. If a MONDO term (MONDO id) is obsoleted, the Mondo term will not appear in the rare disease subset, unregardless of the authoritative rare disease source. The rare disease authoritative source reporting this disease will be contacted to report this obsoletion and its reasons.

Reviewer: ```suggestion
1. If a MONDO term (MONDO id) is obsoleted, the Mondo term will not appear in the rare disease subset, regardless of wether the disease is considered rare by one of the authoritative rare disease sources. The rare disease authoritative source reporting this disease will be contacted to report this obsoletion and its reasons.
```

> 2. If an authoritative rare source obsolete as disease term, or remove a disease from their rare disease list, the corresponding Mondo term will lost its corresponding "rare subset" annotation for this particular source. It should be noted that the rare subset annotation for the other authorities will not be modified unless they also obsolete or remove the corresponding disease from their rare disease list.

Reviewer: ```suggestion
2. If an authoritative rare disease source obsoletes a disease term, or removes a disease from their rare disease list, the corresponding Mondo term will lost its corresponding "rare subset" annotation for this particular source. It should be noted that the rare subset annotation for the other authorities will not be modified unless they also obsolete or remove the corresponding disease from their rare disease list.
```
